### ISSUE TITLE
Handwritten recognition

### ISSUE THREAD

**Issue by ioannexpert** [ioannexpert/nexpertERP#218]
Please create an algorithm based on artificial neural network architecture that recognizez handwritten numbers from 0 to 9.
The neural network architecture should have the following structure:
1. One input layer with 784 neurons representing the number of pixels of the 28x28 black and white image
2. One hidden fully connected layer with the input layer
3. One output layer with 10 neurons representing the classification of the numbers from 0 to 9

Other considerations: 
1. The pictures has pixels with numbers from 0 to 255 one one RGB channel because is black and white. The pixel numbers has to be transformed first into numbers from 0 to1.
2. The picture is a 2D array pixel numbers 28 rows and 28 columns representing picture size. After the transformation of values with numbers from 0 to 1 it has to be transposed into 2D array wit 784 values. 

Use tensor flow dense function to create your architecture using softmax activation method. 

Learn material : https://www.youtube.com/watch?v=iqQgED9vV7k
